packages/headless/src/controllers/core/facets/category-facet/headless-core-category-facet.ts

```diff
@@ -48,15 +47,22 @@ export interface CategoryFacetProps {
   options: CategoryFacetOptions;
 }
 
-/**
- * The `CategoryFacet` headless controller offers a high-level interface for designing a facet UI controller that renders values hierarchically.
- */
-export interface CategoryFacet extends Controller {
+export interface CategoryFacet extends CoreCategoryFacet {
```

Comment:
Should this be moved to `headless-category-facet.ts`?

Reply:
I mostly kept them there because they were there in the first place :sweat: 

Both Product Listing and Search uses it, so I can't really put it in the Search-specific one